packages/map/src/components/Legend/components/Legend.tsx

```diff
@@ -187,30 +187,32 @@ const Legend = () => {
             {(state.visual.additionalCityStyles.some(c => c.label) || state.visual.cityStyleLabel) && (
               <>
                 <hr />
-                {state.visual.cityStyleLabel && (
-                  <div className='container-shapes'>
-                    <svg>
-                      <Group top={state.visual.cityStyle === 'pin' ? 19 : state.visual.cityStyle === 'triangle' ? 13 : 11} left={10}>
-                        {shapes[state.visual.cityStyle.toLowerCase()]}
-                      </Group>
-                    </svg>
-                    <p>{state.visual.cityStyleLabel}</p>
-                  </div>
-                )}
-
-                {state.visual.additionalCityStyles.map(
-                  ({ shape, label }) =>
-                    label && (
-                      <div className='container-shapes'>
-                        <svg>
-                          <Group top={shape === 'Pin' ? 19 : shape === 'Triangle' ? 13 : 11} left={10}>
-                            {shapes[shape.toLowerCase()]}
-                          </Group>
-                        </svg>
-                        <p>{label}</p>
-                      </div>
-                    )
-                )}
+                <div className={legendClasses.div.join(' ') || ''}>
+                  {state.visual.cityStyleLabel && (
+                    <div>
+                      <svg>
+                        <Group top={state.visual.cityStyle === 'pin' ? 19 : state.visual.cityStyle === 'triangle' ? 13 : 11} left={10}>
+                          {cityStyleShapes[state.visual.cityStyle.toLowerCase()]}
+                        </Group>
+                      </svg>
+                      <p>{state.visual.cityStyleLabel}</p>
+                    </div>
+                  )}
+
+                  {state.visual.additionalCityStyles.map(
+                    ({ shape, label }) =>
+                      label && (
+                        <div>
```

Comment:
Similar, do we need an empty div here?

Reply:
We need these because we apply css flex on them to align <p> & svg